src/components/ProjectionError.js

```diff
@@ -4,32 +4,22 @@ import { useTranslation } from 'react-i18next';
 import { Message } from 'semantic-ui-react';
 import styles from './projectionError.module.css';
 
-const requiredFieldsForProjection = [
-  'forestEcoregion',
-  'forestType',
-  'altitudinalZone',
-];
-
 function ProjectionError() {
   const { mapLocation, projectionMode } = useSelector(state => ({
     mapLocation: state.mapLocation,
     projectionMode: state.projectionMode,
   }));
 
   const { t } = useTranslation();
-  const missingFields = requiredFieldsForProjection
-    .filter(field => mapLocation[field] === undefined)
-    .map(field => t(`${field}.label`));
 
-  return projectionMode === 'm' &&
-    mapLocation.coordinate &&
-    missingFields.length >= 1 ? (
+  return projectionMode === 'm' && mapLocation.forestType === '50a' ? (
```

Comment:
This should be handled in a more general way. Invalid forest types should not be hardcoded. We need to catch the error from the project function and handle it there.